### PR TITLE
Preparing Maintenance Release

### PR DESCRIPTION
**Pull request by WardF**
Intermediary branch for v4.6.2.1.  Will merge changes in this branch back upstream.

### CODE REVIEW

**Comment by edhartnett** [comment]
Is there some reason not to call this 4.6.3? Added an extra number to the version is non-standard...

**Comment by WardF** [comment]
It is a maintenance release, and in line with historic releases.

**Comment by WardF** [comment]
Doh, clicked the wrong button, wondered where this had gone.  Upon reflection and also some additional bug fixes, I concede the point and will be recreating this as 4.6.3.  Hopefully to go out tomorrow once the conda issues are addressed.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -7,9 +7,15 @@ This file contains a high-level description of this package's evolution. Release
 
 ## 4.6.3 - TBD
 
-* Update the license from the home-brewed NetCDF license to the standard 3-Clause BSD License.  This change does not result in any new restrictions; it is merely the adoption of a standard, well-known and well-understood license in place of the historic NetCDF license written at Unidata.  This is part of a broader push by Unidata to adopt modern, standardized licensing.  
+## 4.6.2.1 - February 15, 2019
 
+### Maintenance Release
 
+* Update the license from the home-brewed NetCDF license to the standard 3-Clause BSD License.  This change does not result in any new restrictions; it is merely the adoption of a standard, well-known and well-understood license in place of the historic NetCDF license written at Unidata.  This is part of a broader push by Unidata to adopt modern, standardized licensing.
```
Comment (edhartnett) [inline RELEASE_NOTES.md]: Should add:
* Now always write hidden coordinates attribute, which allows faster file opens when present (#1262).
* Some fixes for rename, including fix for renumbering of varids after a rename (#1307), renaming var to dim without coordinate var (#1297).
* Fix of NULL parameter causing segfaults in put_vars functions (#1265).
* Fix of --enable-benchmark benchmark tests (#1121).
* --disable-relax-coordinate-bound option deprecated, and will be removed in the next release. Use of this option can create builds that behave inconsistently on HPC systems. Coord bounds should always be relaxed, as is the current default. (#1010).

File: docs/guide.dox [hunk 2]
```diff
@@ -88,12 +88,8 @@ were written.
 
 Unidata supports the netCDF interfaces for C (see <a
 href="http://www.unidata.ucar.edu/netcdf/docs/" >NetCDF-C User's
-Guide</a>), FORTRAN 77 (see <a
-href="http://www.unidata.ucar.edu/netcdf/documentation/historic/netcdf-f77/index.html#Top"
->NetCDF Fortran 77 Interface Guide</a>), FORTRAN 90 (see <a
-href="http://www.unidata.ucar.edu/netcdf/documentation/historic/netcdf-f90/index.html"
->NetCDF Fortran 90 Interface Guide</a>), and C++ (see <a
-href="http://www.unidata.ucar.edu/netcdf/documentation/historic/cxx4/index.html"
+Guide</a>), Fortran (see <a href="http://www.unidata.ucar.edu/netcdf/fortran/docs/" >NetCDF-Fortran User's Guide</a>)
```
Comment (edhartnett) [inline docs/guide.dox]: Awesome!

**Pull request merged**